gtwrap/pybind_wrapper.py

```diff
@@ -307,7 +338,9 @@ def wrap_namespace(self, namespace):
         if len(namespaces) < len(self.top_module_namespaces):
             for element in namespace.content:
                 if isinstance(element, parser.Include):
-                    includes += ("{}\n".format(element).replace('<', '"').replace('>', '"'))
+                    includes += ("{}\n".format(element).replace('<',
+                                                                '"').replace(
+                                                                    '>', '"'))
```

Comment:
weird autoformat

Reply:
See above comment - I thought this was unintentional.  I see you edited anyway :)